package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@nebula.js/sn-action-button",
   "version": "1.14.3",
-  "description": "Supernova action button",
+  "description": "Button chart for executing actions",
```

Comment:
is it a chart really 🤔

Reply:
you find it under charts in the client, but it is a bit weird I agree. I change it
<img width="408" alt="Screenshot 2021-12-20 at 11 19 22" src="https://user-images.githubusercontent.com/9249820/146752615-bfae3123-7d0f-4903-92ba-a6df81887922.png">
.

Reply:
Good that you bring it up. As I understand that the button in the Charts is not sn-action-button yet, so I found that in the assets folder, the preview.gift does not look right to me. It should be a static image that just looks like in your attached image, the icon image which is left to the Button.

Reply:
aha, I see. Thank you! Then I believe that the gif should be moved to other places for readme, and not in the assets folder which results in being copied into the dist folder to increase the size for nothing.

Reply:
Well all the other repos have images stored in the assets folder. I think the qext file ended up in there and thus the whole folder was copied. But as you wrote in the other comment, unclear if we still need to copy assets. I think we should just not copy it. No reason to have images in the package, just extra size. There are plenty of documentation on github and dev portal